src/rest/Facet/FacetRequest.ts

```diff
@@ -200,6 +200,13 @@ export interface IFacetRequest {
    */
   basePath?: string[];
 
+  /**
+   * Whether to use `basePath` as a filter for the results.
+   *
+   * **Default (Search API):** `true`
```

Comment:
I think we should explain a bit more here. I'd rephrase that, by default, the option is `undefined` and the Coveo platform will decide what will happen. And then if we can explain that the Search API will filter by default on the given facet field/base path.

Reply:
@fbeaudoincoveo copied the doc from the PR on SEARCH-API
Maybe it should be more descriptive?
That file is only the interface for the API request, not the JSUI option that uses that parameter.